docker/Dockerfile

```diff
@@ -36,7 +36,11 @@ RUN echo -e '\n# Activate python virtual environment\nif [ -d /venv/bin ]; then\
     chown -R atlas /venv && \
     python --version --version && \
     python -m pip list && \
-    root --version
+    root --version && \
+    git clone https://github.com/scikit-hep/uproot5.git && \
+    cd uproot5 && \
+    python -m pip install -e . && \
+    cd ..
```

Comment:
No do not install realeases that are not viable stable releases yet. This makes the lock file worthless.

Reply:
I agree. I will revert this as soon as there is a way to test on branch built image. We also need to get that image selection dropbox autopopulated with all the named images in the repo.